src/java/boa/compiler/SymbolTable.java

```diff
@@ -379,7 +380,9 @@ public static void resetTypeMap() {
 		types.put("float", new BoaFloat());
 		types.put("time", new BoaTime());
 		types.put("string", new BoaString());
-
+		types.put("LinearRegression", new BoaLinearRegression());
+		types.put("Model", new BoaModel());
+		types.put("tuple", new BoaTuple());
```

Comment:
why is this here? `tuple` isnt a type (it is, but not with the keyword tuple)

Reply:
I have removed the line 385.